.github/workflows/go-format.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     steps:
       - uses: actions/setup-go@v2
         with:
-          go-version: '1.18'
+          go-version: '1.23'
```

Comment:
Changed due to it failing

https://github.com/applejag/typ/actions/runs/11437566167/job/31817322585#step:3:9